packages/graphql-transformers-e2e-tests/package.json

```diff
@@ -22,6 +22,7 @@
     "build-tests": "yarn tsc --build tsconfig.tests.json"
   },
   "dependencies": {
+    "@aws-amplify/graphql-auth-transformer": "^0.9.2",
```

Comment:
Can be 0.9.2 since lerna version will manage bumps.

Reply:
updated